README.md

```diff
@@ -194,7 +194,7 @@ to see the current progress of all translations.
 - [kb](https://github.com/gnebbia/kb)
   is a minimalist command-line knowledge base manager.
   kb can be used to organize your notes and cheatsheets in a minimalist
-  and clean way. It supports also non-text files.
+  and clean way. It supports  non-text files also.
```

Comment:
double space here
```suggestion
  and clean way. It supports non-text files also.
```

Reply:
The also would go at the beginning on the sentence otherwise its less formal

Reply:
I also think it rather should be
```suggestion
  and clean way. It also supports non-text files.
```

Reply:
@marchersimon's suggestion reads most naturally to me.

Reply:
Leaving unresolved on purpose.